src/dune_digest/md5_stubs.c

```diff
@@ -0,0 +1,57 @@
+#include <caml/mlvalues.h>
+#ifdef _WIN32
+#include <caml/fail.h>
+
+CAMLprim value dune_md5_fd(value v_fd) {
+  caml_failwith("not supported on windows");
```

Comment:
Unless I am missing something, there is nothing Unix-specific in this code, is there?

Reply:
I gave it a try (I didn't actually test the library; I don't know if it is used anywhere yet...)

Reply:
The library `dune_digest`? It's used by dune to generate the digests so it's definitely used.

Reply:
Ah right, so then we will see if it is working well in the CI :)